src/legacy/core_plugins/kibana/public/visualize/listing/visualize_listing.js

```diff
@@ -94,15 +94,18 @@ export function VisualizeListingController($injector, createNewVis) {
       });
   };
 
-  this.deleteSelectedItems = function deleteSelectedItems(selectedIds) {
-    return visualizationService.delete(selectedIds)
-      .catch(error => {
-        toastNotifications.addError(error, {
-          title: i18n.translate('kbn.visualize.visualizeListingDeleteErrorTitle', {
-            defaultMessage: 'Error deleting visualization',
-          }),
-        });
+  this.deleteSelectedItems = function deleteSelectedItems(_ids, selectedItems) {
```

Comment:
It's the current signature, so removing it would require changing all callers. (We're not the only caller, and there may be 3rd party plugins using it, too.) So, I only made backward-compatible changes.